src/TraceEvent/Computers/TraceManagedProcess.cs

```diff
@@ -522,9 +521,11 @@ internal static void SetupCallbacks(TraceEventDispatcher source)
                             mang.GC.m_stats.suspendThreadIDBGC = data.ThreadID;
                             break;
                         default:
-                            mang.GC.m_stats.suspendThreadIDOther = data.ThreadID;
```

Comment:
> mang.GC.m_stats.suspendThreadIDOther = data.ThreadID; [](start = 27, length = 54)

when you showed me the code I didn't search for the usage of suspendThreadIDOther but now I see there's only one usage and since you are not assigning a value to suspendThreadIDOther anymore that only usage will never be used; so might as well get rid of it all together. I think when I added this there was some peculiar thing wrt GCStart being fired multiple times (when it shouldn't be) but honestly I don't remember the detail. 

otherwise LGTM. thanks for fixing!

Reply:
oops! I meant to revert this line too so that the IDOther would still get initialized in this code. Would you prefer that or you still want me to eliminate it?

Reply:
ahh...then I would prefer to revert :)

Reply:
Should be fixed now. I also needed to add a check to EERestartStop to ensure that we clear threadIDOther since it no longer reaches the bottom of the function as it did before.